app/src/test/java/org/simple/clinic/registration/confirmpin/RegistrationConfirmPinScreenControllerTest.kt

```diff
@@ -83,11 +88,16 @@ class RegistrationConfirmPinScreenControllerTest {
     setupController()
     uiEvents.onNext(RegistrationConfirmPinTextChanged(invalidConfirmationPin))
     uiEvents.onNext(RegistrationConfirmPinDoneClicked())
+
+    verify(screen).showPinMismatchError()
+    clearInvocations(screen)
```

Comment:
Can you tell me what would `clearInvocations(screen)` do?

Reply:
Since it's a mock, it retains a history of all the interactions with the instance that has happened so far. `clearInvocations` will clear these interactions.

This lets us verify that any future interactions have happened only due to a result of events that we triggered after clearing the invocations.